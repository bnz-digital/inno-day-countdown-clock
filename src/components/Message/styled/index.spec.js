import React from 'react'
import { shallow } from 'enzyme'

import StyledMessage from '.'

describe('components:Message:styled', () => {
  it('renders <StyledMessage/> correctly', () => {
    expect(toJson(shallow(<StyledMessage />))).toMatchSnapshot()
  })
})
