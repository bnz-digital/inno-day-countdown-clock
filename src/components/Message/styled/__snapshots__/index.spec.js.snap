// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:Message:styled renders <StyledMessage/> correctly 1`] = `
.c0 {
  font-size: 4vw;
  grid-area: message;
  place-self: center;
  padding-top: 10vh;
}

<div
  className="c0"
/>
`;
